### PR TITLE
Planning: plot the output path on dreamview.

### PR DESCRIPTION
**Pull request by panjiacheng**

### CODE REVIEW

File: modules/planning/common/planning_context.h [hunk 1]
```diff
@@ -104,7 +104,8 @@ class PlanningContext {
 
   static const LaneFollowInfo& lane_follow_info() { return lane_follow_info_; }
 
-  static LaneFollowInfo* mutable_lane_follow_info() { return &lane_follow_info_; }
+  static LaneFollowInfo* mutable_lane_follow_info() {
+      return &lane_follow_info_; }
```
Comment (xiaoxq) [inline modules/planning/common/planning_context.h]: No need to change

File: modules/planning/tasks/deciders/path_assessment_decider.cc [hunk 2]
```diff
@@ -69,13 +73,16 @@ Status PathAssessmentDecider::Process(
     const std::string msg = "Neither regular nor fallback path is valid.";
     AERROR << msg;
     return Status(ErrorCode::PLANNING_ERROR, msg);
+  } else {
+    ADEBUG << "There are " << valid_path_data.size() << " valid path data.";
```
Comment (xiaoxq) [inline modules/planning/tasks/deciders/path_assessment_decider.cc]: No need to put in `else`

File: modules/planning/common/planning_context.h (hunk 1 above)
Comment (panjiacheng) [reply modules/planning/common/planning_context.h]: line was too long and violated lint.

File: modules/planning/tasks/deciders/path_assessment_decider.cc (hunk 2 above)
Comment (panjiacheng) [reply modules/planning/tasks/deciders/path_assessment_decider.cc]: fixed.

**Pull request merged**